packages/components/.babelrc.json

```diff
@@ -0,0 +1,16 @@
+{
+    "sourceType": "unambiguous",
+    "presets": [
+        [
+            "@babel/preset-env",
+            {
+                "targets": {
+                    "chrome": 100
```

Comment:
In Suite, we do not use `@babel/preset-env`, so I guess we should not use it here

Reply:
It should use the same config as we have in `base.webpack.config.js`

Reply:
True, this got added automatically by `npx storybook@latest upgrade` I believe